master/buildbot/status/web/mybuilds.py

```diff
@@ -40,6 +42,12 @@ def content(self, req, cxt):
         template.autoescape = True
         defer.returnValue(template.render(**cxt))
 
+    @staticmethod
+    def add_css_classes_to_results(builds):
```

Comment:
I think it can be a util function and I am sure that it should return a copy of builds (not modify passed parameters)

Reply:
I've added a docstring to function and next, I've moved function and tests to new files (again `util.build` and `test_util_build`). 
But we will have to merge code during creation `test/staging` and `staging`